description.md

```diff
@@ -17,6 +17,121 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Quizzes
 
+#### Definition
+
+Quizzes are questions that verify specific information and must be verifiable automatically.
+Therefore, multiple-choice questions are regarded as quizzes, while open-ended ones are not.
+Quizzes may be part of practical sessions, lectures or standalone as forms of evaluation.
+
+#### Purpose
+
+Quizzes provide a means to continuously evaluate students.
+This creates a positive feedback loop between trainers and trainees whereby both can track the students' progress in real time.
+
+Furthermore, quizzes serve as practice items for students to solve at their own pace.
+They may also be used for evaluation items that contribute to the students' overall grade.
+
+Another desired aspect is that quizzes be easily extractable and convertible between formats, as described [below](#deployment), and repurposed by other trainers.
+
+Despite all of their strengths, quizzes also come with a few shortcomings.
+One such downside is the binary nature of the feedback loop.
+Because multiple-choice answers may be either correct or wrong, there is little room for nuance regarding the feedback that students receive upon answering incorrectly.
+Similarly, quizzes cannot be easily tailored and personalized for the individual needs of each student.
+In addition, non-technical fields of study may find little benefit in quizzes as they rely more on arguments essays rather than concrete, technical information.
+
```

Comment:
Yes, I would also link it to the last sentence. Something along the lines of: "However, even for these fields, quizzes may be used to evaluate concrete knowledge of specific information".

Reply:
I'm not sure I understand this discussion:  in SmileyTutor (former tutor-web) we allocate different drill items (quiz questions) depending on student ability, so allocation depends on where they are in the learning process. This qualifies as personalised learning, yet using multiple-choice drill items. Note that these are not just ordinary quizzes as in an exam, but individual multiple choice questions (=items in item theory).

So I don't see how this is correct: "Similarly, quizzes cannot be easily tailored and personalized for the individual needs of each student."

Next, this "beause multiple-choice answers may be either correct or wrong, there is little room for nuance regarding the feedback that students receive upon answering incorrectly." --- in SmileyTutor we normally have a detailed explanation which the student sees once they have submitted the answer... In principle one could have different explanations depending on which incorrect answer is submitted. So how is this a problem with multiple-choice??

Reply:
@gstefans The discussion is more about non-technical fields such as philosophy, psychology etc. where having quizzes is not ideal. But this could apply to mathematics also: you have much more insights on the understanding of a student if you evaluate how the student got to the result than if you just evaluate the answer.

Maybe this is not well phrased, but we want to highlight that multiple choice questions may not be ideal in specific scenarios.

Reply:
Right. I still think some care in wording is needed:

Questions with open-ended answer options certainly have many benefits over items with multiple-choice answers. But it is not factually correct that there is no flexibility in providing personalised learning with multiple-choice items. 

It is also a common claim that multiple-choice items only provide for rote learning, but this is also not factually correct - we have papers demonstrating that appropriate use of such items in fact provides transferable knowledge...

Reply:
@teodutu please rephrase this so that we do not have any factually incorrect statements.

Reply:
> there is little room for nuance regarding the feedback that students receive upon answering incorrectly

I meant to say this is a con of automatic grading. It's difficult to find the exact step that the student got wrong and insist on that.